Exercise_6/Exercise_6.ipynb

```diff
@@ -180,33 +180,69 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "278234da",
+   "metadata": {},
+   "source": [
+    "Please open a terminal and go to the directory you get executing the cell above.\n",
```

Comment:
I cannot open a terminal like this. It looks like this for me:
![open_terminal](https://user-images.githubusercontent.com/27766978/161551321-e47935c8-db85-4614-9a3c-caa4534aad25.png)

Reply:
You don't open a terminal on the screenshot but start a new kernel that uses bash.

We want the users to open a real terminal.

The screenshot is taken from the IAC Jupyter Hub, so that's the view the participants will have in the end.

Do you understand?